include/board.c

```diff
@@ -30,4 +30,52 @@ void print_board(struct Board *board) {
         printf("\n");
     }
     printf("\n");
+}
+
+int check_draw(struct Board *board)
+{
+    for (int row = 0; row < 3; row++) 
+    {
+        for (int col = 0; col < 3; col++)
+        {
+           if(board->board[row][col] >= '1' && board->board[row][col] <= '9'){
```

Comment:
What is this line of code doing? How is it checking for a draw?

Reply:
this condition checking if any digit from 1-9 is present in the board loop will return 0,which is the condition for no draw and if none of the digit in range of 1-9 is present in the board (like every index of board is filled with either X or O ) the condition will be false and it will return 1

Reply:
The board (both visual and the `board` array) will be filled with empty strings at the start, not with 1-9 numbers. The UI won't look good if the board is filled with numbers.